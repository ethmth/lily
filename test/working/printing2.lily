print(4, 53.2)