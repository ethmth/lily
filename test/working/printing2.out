4 53.200000